lib/elixir/lib/gen_server.ex

```diff
@@ -418,6 +418,11 @@ defmodule GenServer do
 
   If this callback is not implemented, the default implementation by
   `use GenServer` will return `{:noreply, state}`.
+
+  The default implementation catches all messages. If you override it,
```

Comment:
Thank you for your reply! It seems we reached a wrong conclusion in the Forum topic then.

I'll quote your message there.

I guess this means that this PR can be closed :wink: .

Thank you!

~W-M/Qqwy